omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskAction.java

```diff
@@ -53,5 +53,9 @@ public Action getAction() {
 	public boolean isAllTasks() {
 		return allTasks;
 	}
-	
+
+	public void setAllModules(boolean allTasks) {
+		this.allTasks = allTasks;
```

Comment:
Can't we just drop the all tasks property? Of what use is it?

Reply:
Here I have some SchedulerService functions for allTasks, So if the request contains `allTasks = true `then I will call that function to perform the actions for all tasks (like resheduleAllTasks). 
If the request doesn't contain `allTasks` property, then it will be false (initialization default value)

Reply:
Do you mean is someone sends a list of tasks to be acted upon, then need to include the allTasks flag? If allTasks = false, does it mean that the action is just some of the passed in tasks but not others?

Reply:
For Example, If the system has 5 tasks and he sent a list of 3 tasks (without allTasks property), then the resource will do that respected action for only that 3 tasks.

If he wants to perform that action for all 5 tasks, then he can just send action type with `allTasks=true` property. Then resource will not think about the list of tasks in the request. 
if will perform the required action for all tasks on the server.

Reply:
Thanks for the clarification. 😊